os-eap-launch/module.yaml

```diff
@@ -5,3 +5,14 @@ description: Legacy os-eap-launch script package.
 execute:
 - script: configure.sh
   user: '185'
+packages:
+      repositories:
+          - jboss-os
+      install:
+          - hostname
+envs:
+    - name: "DEFAULT_ADMIN_USERNAME"
```

Comment:
I think this can probably be removed from the other product -launch modules, since I think they all include os-eap-launch, which should be common.  In general, the environment variables listed should be used by the module within which they are contained, e.g. if it's not used in os-jdg-launch, it shouldn't be there.  Keep in mind, one of the goals of this effort is to remove duplication.

Reply:
I've removed this from all modules except os-eap-launch

I've added a module depenency on os-eap-launch to each module I removed it from